topi/python/topi/vision/sort.py

```diff
@@ -0,0 +1,88 @@
+"""Argsort operator"""
```

Comment:
Can we move sort.py out of vision folder?

Reply:
Not sure what is the best location for sort

Reply:
Maybe we can move sort.py to topi root directory, to make it another standalone category?

Reply:
Let's just move to `topi/python/topi/sort.py` since `sort` is a basic op.